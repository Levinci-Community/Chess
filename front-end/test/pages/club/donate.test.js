import React from "react";
import { render } from "@testing-library/react";
import DonatePage from "../../../src/pages/club/donate";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({
    t: (key) => key,
  }),
}));

test("renders DonatePage snapshot", () => {
    const { container } = render(
      <Router>
        <DonatePage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
