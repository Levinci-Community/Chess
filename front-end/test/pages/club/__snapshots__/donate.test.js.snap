// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DonatePage snapshot 1`] = `
<div>
  <div
    class="css-1ja7hik"
  >
    <div
      class="css-k008qs"
    >
      <img
        alt="chess1"
        class="chakra-image css-hdsgkm"
        src="[object Object]"
      />
      <div
        class="css-17xejub"
      />
      <img
        alt="chess2"
        class="chakra-image css-1vn57wq"
        src="[object Object]"
      />
      <div
        class="css-17xejub"
      />
      <img
        alt="chess3"
        class="chakra-image css-6jwltp"
        src="[object Object]"
      />
      <div
        class="css-17xejub"
      />
      <img
        alt="chess4"
        class="chakra-image css-hdsgkm"
        src="[object Object]"
      />
    </div>
    <div
      class="css-1pstfjo"
    >
      <img
        alt="chess5"
        class="chakra-image css-u9uoi6"
        src="[object Object]"
      />
      <div
        class="css-17xejub"
      />
      <h1
        class="chakra-heading css-1eth8b1"
      >
        club.support_ute_chess_club
      </h1>
      <div
        class="css-70qvj9"
      >
        <img
          alt="club.support_ute_chess_club"
          class="chakra-image css-1p3fvi7"
          src="undefined/images/donate"
        />
        <div
          class="css-1isemmb"
        >
          <p
            class="chakra-text css-1fhby5u"
          >
            club.card_bank
            : Vietcombank
          </p>
          <p
            class="chakra-text css-1fhby5u"
          >
            club.card_number
            : 022 6868 6868 6868
          </p>
          <p
            class="chakra-text css-1fhby5u"
          >
            club.card_name
            : CLB Cá»œ SPKT
          </p>
        </div>
      </div>
      <div
        class="css-17xejub"
      />
      <img
        alt="chess6"
        class="chakra-image css-u9uoi6"
        src="[object Object]"
      />
    </div>
    <div
      class="css-ltqa49"
    >
      <img
        alt="chess7"
        class="chakra-image css-1c55xed"
        src="[object Object]"
      />
      <div
        class="css-17xejub"
      />
      <div
        class="css-1mki2y5"
      >
        <p
          class="chakra-text css-19nm251"
        >
          club.your_donate_help_us_providing
        </p>
        <p
          class="chakra-text css-1mwnirl"
        >
          club.thank_you
        </p>
      </div>
      <div
        class="css-17xejub"
      />
      <img
        class="chakra-image css-1067xm8"
        src="[object Object]"
      />
    </div>
  </div>
</div>
`;
